### ISSUE TITLE
Product of output shape dimensions does not match product of argument shape dimensions (Raspbian, MYRIAD)

### ISSUE THREAD

**Issue by nyckyta** [openvinotoolkit/openvino#482]
Hello!

I have installed openvino toolkit (2.2 release version) on raspberry pi 3b+, but unfortunately nor test example neither my own do not work and fail printing this log:

``` 
[ INFO ] InferenceEngine: 
        API version ............ 2.1
        Build .................. 42025
        Description ....... API
Parsing input parameters
[ INFO ] Files were added: 1
[ INFO ]     /home/pi/img.jpg
[ INFO ] Loading Inference Engine
[ INFO ] Device info: 
        MYRIAD
        myriadPlugin version ......... 2.1
        Build ........... 42025
[ INFO ] Loading network files:
        face-detection-adas-0001.xml
        face-detection-adas-0001.bin
[ ERROR ] Check 'input_shape_product.get_length() == shape_size(m_output_shape)' failed at /teamcity/work/scoring_engine_build/releases_2020_2/ngraph/src/ngraph/op/reshape.cpp:100:
While validating node 'v0::Reshape Reshape_971(PriorBox_744[0]:f32{2,3899752448}) -> (dynamic?)':
Product of output shape dimensions does not match product of argument shape dimensions (output shape: Shape{1, 2, 3899752448}, argument shape: {2,3899752448}).
```

Have I made some mistakes with installation? Should version to be 2.2?
**3899752448** - that number from log different after each attempt.

**Comment by leavim** [comment]
Hi @nyckyta , thanks for the information. We are aware of this issue and are able to reproduce, currently investigating the problem and looking for a fix. 

I'll let you know in case there is a workaround. My suggestion would be to try and use a previous release if possible, since this problem seems to affect the 2020.2 release. 

Regards,
Luis

**Comment by nyckyta** [comment]
Thank you @leavim.

**Comment by roccopietrini** [comment]
Hi, same here. I just fresh installed openvino toolkit on raspberry pi4 following this guide: [https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_raspbian.html](url) I cannot run neither the compiled cpp example nor the python opencv one with the aforementioned error. 
In addition following this tutorial [https://www.pyimagesearch.com/2019/04/08/openvino-opencv-and-movidius-ncs-on-the-raspberry-pi/](url) I get a segfault on net.forward..
Any news on this ?

**Comment by natxopedreira** [comment]
This is related
https://github.com/openvinotoolkit/openvino/issues/411

Last releases models do not work

2020 models r1 do not work
2020 models r2 do not work
2020 models r3 do not work

You need to downgrade to 2019 r3 models to make it work, its a shame because 2020 has various face models that you can not use

**Comment by avitial** [comment]
This issue has been addressed, you should be able to use v2020.4 models with v2020.4 OpenVINO toolkit. Please upgrade your installation to this version and let us know if you face any issues.

Closing this as it has been addressed, feel free to reopen and ask additional questions related to this topic if issue persists.
